.github/workflows/build-and-analyze.yml

```diff
@@ -1,34 +1,30 @@
 name: Code test and analysis
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
     types: [opened, synchronize, reopened]
   workflow_dispatch:
-jobs:   
+jobs:
   build-and-test:
     name: Build and Test
-    if: ((github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false) || github.event_name == 'push') && github.repository_owner == 'Altinn' && github.actor != 'dependabot[bot]' 
+    if: ((github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false) || github.event_name == 'push') && github.repository_owner == 'Altinn' && github.actor != 'dependabot[bot]'
```

Comment:
Vi bruker ikke dependabot lenger så de actor sjekkene kan vel fjernes.
Sånn sett usikker på hvorfor man har unntak for build & test for dependabot uansett. Burde jo kjøres for alle patch PRs